module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -542,6 +542,42 @@ protected function fetchUserWithCql($query)
         return count($json->users) === 1 ? $json->users[0] : null;
     }
 
+    /**
+     * Helper function to retrieve paged results from FOLIO API
+     *
+     * @param string $responseKey Key containing values to collect in response
+     * @param string $interface   FOLIO api interface to call
+     * @param array  $query       CQL query
+     *
+     * @return array
+     */
+    protected function getPagedResults($responseKey, $interface, $query = [])
+    {
+        $fullResponse = [];
+        // set artificially low to test
```

Comment:
Is this comment necessary? (Is this a placeholder to remind you to do something, or just a comment about how you can test? If the latter, does it need to be said?)